src/tests/mod.rs

```diff
@@ -9,19 +9,33 @@ use std::net::{TcpListener, UdpSocket};
 use crate::interface;
 use crate::safeposix::{cage::*, filesystem::*};
 
+#[allow(unused_parens)]
```

Comment:
Why do we need this?

Reply:
we have a couple of places that through - "warning: unnecessary parentheses around assigned value". I think it was used to suppress such warning.